snorkel/slicing/modules/slice_combiner.py

```diff
@@ -88,10 +90,15 @@ def forward(  # type:ignore
             flow_dict, self.slice_pred_key
         )
 
+        if predictor_outputs[0].shape[1] > 2:
+            raise NotImplementedError(
+                "SliceCombiner does not support multiclass labels."
+            )
+
         predictor_confidences = torch.cat(
             [
-                # Compute the "confidence" using the max score across classes
-                torch.max(F.softmax(output, dim=1), dim=1)[0].unsqueeze(1)
+                # Compute the "confidence" using probability score of the positive class
+                F.softmax(output, dim=1)[:, 0].unsqueeze(-1)
```

Comment:
yea, we could. by default, the convention of our binary heads are 2-dimensional, so a softmax over the 2 dims is more natural than a sigmoid over just the first dimension.

ideally, would be nice down the line to have 1-dimensional outputs + a sigmoid—but for consistency, the current convention shouldn't hurt.

Reply:
idk if it's more natural...i agree we should leave as 2-dim so that there's no major overhaul when multiclass is added. however, it's more legible and faster to just do `torch.sigmoid(output[:, 0]).unsqueeze(-1)`